docs/source/user_guide/config.rst

```diff
@@ -662,60 +662,62 @@ property.
-| `sidebar_mode`            | `FIFTYONE_APP_SIDEBAR_MODE`            | `fast`                      | The default loading behavior of the App's sidebar. Supported values are                   |
-|                           |                                        |                             | `{"fast", "all", "best", "disabled"}`. See :ref:`this section <app-sidebar-mode>`         |
-|                           |                                        |                             | more details.                                                                             |
-+---------------------------+----------------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
-| `theme`                   | `FIFTYONE_APP_THEME`                   | `"browser"`                 | The default theme to use in the App. Supported values are `{"browser", "dark", "light"}`. |
-|                           |                                        |                             | If `"browser"`, your current theme will be persisted in your browser's storage.           |
-+---------------------------+----------------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
-| `use_frame_number`        | `FIFTYONE_APP_USE_FRAME_NUMBER`        | `False`                     | Whether to use the frame number instead of a timestamp in the expanded sample view. Only  |
-|                           |                                        |                             | applicable to video samples.                                                              |
-+---------------------------+----------------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
-| `plugins`                 | N/A                                    | `{}`                        | A dict of plugin configurations. See :ref:`this section <configuring-plugins>` for        |
-|                           |                                        |                             | details.                                                                                  |
-+---------------------------+----------------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
++----------------------------+-----------------------------------------+---------------+--------------------------------------------------------------------------------------------+
+| Config field               | Environment variable                    | Default value | Description                                                                                |
```

Comment:
FYI- I reverted the `| -> ||` change in the table columns because it didn't render correctly in the built docs